app/styles/components/_searchbox.scss

```diff
@@ -21,7 +21,7 @@
         overflow: show;
         min-width: 220px;
     }
-    button[type=search] {
+    button[type=search], span[type=search] {
```

Comment:
This button isn't in the structure anymore so the style can be removed.  I also think span should probably get a class of search instead of a type which I think it just for buttons.